packages/opentelemetry-basic-tracer/src/types.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ScopeManager } from '@opentelemetry/scope-base';
+import {
+  Attributes,
+  BinaryFormat,
+  HttpTextFormat,
+  Logger,
+  Sampler,
+} from '@opentelemetry/types';
+
+/**
+ * BasicTracerConfig provides an interface for configuring a Basic Tracer.
+ */
+export interface BasicTracerConfig {
```

Comment:
(Maybe later) Should we move `BasicTracerConfig` in `opentelemetry-types`?

Reply:
I think the config can be very different per tracers, but we will see. Let's take a look later.